docs/releases/1.11.txt

```diff
@@ -397,6 +397,8 @@ Tests
 * Added support for :meth:`python:unittest.TestCase.subTest`’s when using the
   :option:`test --parallel` option.
 
+* Tests now run checks at the start of the test run, as they did in Django 1.7.
```

Comment:
I think "as they did in Django 1.7" needn't be mentioned at this point. For third-party test runners, it's probably also useful to mention exactly where the check are run from.